.env.example

```diff
@@ -14,6 +14,7 @@ ETH_STATS_ADDRESS=stats.execution.mainnet.lukso.network
 LUKSO_MAINNET_NETWORK_ID=42
 
 PRYSM_BEACON_VERSION=v4.2.1
+PRYSM_VALIDATOR_VERSION=v4.2.1
```

Comment:
I also made the prysm validator version configurable via `.env`. Be aware that I do not know the motivation to omit this version from `.env`. **This bumps prysm validator version from `v4.0.3` to `v4.2.1`!**

Reply:
This is a good change. We have adjusted it at the `develop` branch, but we can leave it here at the main too.